app/Http/Controllers/OmNomController.php

```diff
@@ -251,7 +251,20 @@ public function buy(Request $request, string $store_slug)
             $result->message = '';
 
             if ($user->show_omnomcom_total) {
-                $result->message = sprintf('You have spent a total of <strong>€%0.2f</strong>', OrderLine::query()->where('user_id', $user->id)->where('created_at', 'LIKE', sprintf('%s %%', date('Y-m-d')))->sum('total_price'));
+                $categories = collect($stores)->pluck('categories')->flatten()->unique();
+
+                $totalSpent = OrderLine::query()
+                    ->where('user_id', $user->id)
+                    ->where('created_at', 'LIKE', sprintf('%s %%', date('Y-m-d')))
+                    ->whereHas('product.categories', function ($query) use ($categories) {
+                        $query->whereIn('categories.id', $categories);
```

Comment:
categories.id fails, because categories is not a table in the db. It should probably be something like product_categories, but look at the model what it should be.

Reply:
yup its called `product_categories` updating it.